css/style.css

```diff
@@ -0,0 +1,33 @@
+@import url('https://fonts.googleapis.com/css2?family=Bebas+Neue&family=Roboto&display=swap');
+
```

Comment:
Move your HTML scripts to the top of the page.

Reply:
- [ ]  I think here the lines you meant to reference were actually [HTML scripts](https://github.com/melaniesigrid/past_project_review/pull/2/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R73-R83). Kindly make sure your comments point to the right lines so the person being reviewed can take out the most from the review with minimal effort. The smoother the review process, the easier it is to foster a culture of learning and sharing.
- [ ]   Also, remember to try to be more suggestive rather than commanding when giving a review. We don't want the receiver to feel like we are giving them orders so they feel open to accepting our feedback. Here instead of saying
> Move your HTML scripts to the top of the page

You can maybe phrase it like

> I think it would be better to move your ...

You can reference the [How to Communicate](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-communicate) section for a more detailed guide

- [ ]  What do you think about linking the user to a doc like [script-async-defer](https://javascript.info/script-async-defer) here?. There should always be a purpose behind why we are requesting the change and linking to documents and giving a reason for our desired change always makes the code review process seem more meaningful :100:

Reply:
- [ ]  I think here the lines you meant to reference were actually [HTML scripts](https://github.com/melaniesigrid/past_project_review/pull/2/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R73-R83). Kindly make sure your comments point to the right lines so the person being reviewed can take out the most from the review with minimal effort. The smoother the review process, the easier it is to foster a culture of learning and sharing.
- [ ]   Also, remember to try to be more suggestive rather than commanding when giving a review. We don't want the receiver to feel like we are giving them orders so they feel open to accepting our feedback. Here instead of saying
> Move your HTML scripts to the top of the page

You can maybe phrase it like

> I think it would be better to move your ...

You can reference the [How to Communicate](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-communicate) section for a more detailed guide

- [ ]  What do you think about linking the user to a doc like [script-async-defer](https://javascript.info/script-async-defer) here?. There should always be a purpose behind why we are requesting the change and linking to documents and giving a reason for our desired change always makes the code review process seem more meaningful :100: